### PR TITLE
Exposes playback speed method

### PR DESCRIPTION
**Pull request by kul3r4**
#### WHAT
Currently playback speed can be set but cannot be read from using the interface

#### WHY
So that the playback speed can be read and the value can be represented in the UI

#### HOW
Add a get method to the interface

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt [hunk 1]
```diff
@@ -198,4 +198,9 @@ public interface PlayerRepository {
      * Set the playback speed.
      */
     public fun setPlaybackSpeed(speed: Float)
+
+    /**
+     * Returns the current playback speed.
+     */
+    public fun getPlaybackSpeed(): Float
```
Comment (luizgrp) [inline media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt:205]: what about adding [playbackSpeed](https://github.com/google/horologist/blob/8482a1f297a5d3c95151b5c2ff3f65d0f934a221/media-data/src/main/java/com/google/android/horologist/media/data/repository/PlayerRepositoryImpl.kt#L89) property to PlayerRepository like we do with these [other](https://github.com/google/horologist/blob/8482a1f297a5d3c95151b5c2ff3f65d0f934a221/media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt#L33-L68) properties?

Comment (kul3r4) [reply media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt]: Thank you, I updated the code

File: media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt [hunk 2]
```diff
@@ -67,6 +67,11 @@ public interface PlayerRepository {
      */
     public val seekForwardIncrement: StateFlow<Duration?>
 
+    /**
+     * Returns the current [playbackSpeed].
+     */
+    public val playbackSpeed: StateFlow<Float?>
```
Comment (luizgrp) [inline media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt:73]: do we want to allow `null` to be returned? the implementation is defined as non-nullable (`StateFlow<Float>`), just wondering if there is a use case for this

Comment (kul3r4) [reply media/src/main/java/com/google/android/horologist/media/repository/PlayerRepository.kt]: agree, removed null

**Review by luizgrp — APPROVED** [review]
LGTM!

**Pull request merged**